kml/edit-kml-ground-overlay/src/main/java/com/esri/samples/edit_kml_ground_overlay/EditKMLGroundOverlaySample.java

```diff
@@ -0,0 +1,162 @@
+import javafx.scene.layout.BackgroundFill;
+import javafx.scene.layout.CornerRadii;
+import javafx.scene.layout.StackPane;
+import javafx.scene.layout.VBox;
+import javafx.scene.paint.Color;
+import javafx.scene.paint.Paint;
+import javafx.stage.Stage;
+
+import com.esri.arcgisruntime.geometry.Envelope;
+import com.esri.arcgisruntime.geometry.SpatialReferences;
+import com.esri.arcgisruntime.layers.KmlLayer;
+import com.esri.arcgisruntime.mapping.ArcGISScene;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.Viewpoint;
+import com.esri.arcgisruntime.mapping.view.Camera;
+import com.esri.arcgisruntime.mapping.view.SceneView;
+import com.esri.arcgisruntime.ogc.kml.KmlDataset;
+import com.esri.arcgisruntime.ogc.kml.KmlGroundOverlay;
+import com.esri.arcgisruntime.ogc.kml.KmlIcon;
+import com.esri.arcgisruntime.symbology.ColorUtil;
+
+public class EditKMLGroundOverlaySample extends Application {
+
+  private KmlGroundOverlay kmlGroundOverlay;
+  private SceneView sceneView;
+  private Slider slider;
+
+  @Override
+  public void start(Stage stage) {
+
+    try {
+      // create stack pane and application scene
+      StackPane stackPane = new StackPane();
+      Scene fxScene = new Scene(stackPane);
+      fxScene.getStylesheets().add(getClass().getResource("/style.css").toExternalForm());
+
+      // set title, size and add application scene to stage
+      stage.setTitle("Edit KML Ground Overlay Sample");
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(fxScene);
+      stage.show();
+
+      // create a scene and add it to the scene view
+      ArcGISScene scene = new ArcGISScene(Basemap.createImagery());
+      sceneView = new SceneView();
+      sceneView.setArcGISScene(scene);
+
+      // create a geometry for the ground overlay
+      Envelope overlayGeometry = new Envelope(-123.066227926904, 44.04736963555683, -123.0796942287304, 44.03878298600624, SpatialReferences.getWgs84());
+
+      // create a KML icon for the overlay image
+      String overlayImageURI = new File("./samples-data/images/1944.jpg").getAbsolutePath();
+      KmlIcon overlayImage = new KmlIcon(overlayImageURI);
+
+      // create the KML ground overlay
+      kmlGroundOverlay = new KmlGroundOverlay(overlayGeometry, overlayImage);
+
+      // set the rotation of the ground overlay
+      kmlGroundOverlay.setRotation(-3.046024799346924);
+
+      // create a KML dataset with the ground overlay as the root node
+      KmlDataset kmlDataset = new KmlDataset(kmlGroundOverlay);
+
+      // create a KML layer using the KML dataset
+      KmlLayer kmlLayer = new KmlLayer(kmlDataset);
+
+      // add the KML layer to the scene view
+      sceneView.getArcGISScene().getOperationalLayers().add(kmlLayer);
+
+      // set the viewpoint to the ground overlay
+      sceneView.setViewpoint(new Viewpoint(kmlGroundOverlay.getExtent(), new Camera(kmlGroundOverlay.getGeometry().getExtent().getCenter(), 1250, 45, 60, 0)));
```

Comment:
Can you reuse `overlayGeometry` instead of `kmlGroundOverlay.getExtent()` and `kmlGroundOverlay.getGeometry().getExtent()`?